src/Components/Search/SearchSection.tsx

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import SearchCard from "./SearchCard";
+
+interface SearchSectionProps {
+  title: string;
+  items: { title: string }[];
+  buttonText: string;
+}
+
+export default function SearchSection({
+  title,
+  items,
+  buttonText,
+}: SearchSectionProps) {
+  return (
+    <section className="mb-8">
+      <div className="flex justify-between items-center mb-2 px-2 mx-44">
```

Comment:
상위 div에 max-w-screen-lg가 생김에 따라 mx-44는 필요가 없을 것 같습니다
```suggestion
      <div className="flex justify-between items-center mb-2 px-2">
```